src/pip/_internal/cache.py

```diff
@@ -28,7 +33,12 @@ class Cache(object):
             ('binary' and 'source' are the only allowed values)
     """
 
-    def __init__(self, cache_dir, format_control, allowed_formats):
+    def __init__(self,
+                 cache_dir,       # type: str
+                 format_control,  # type: FormatControl
+                 allowed_formats  # type: Set[str]
+                 ):
+        # type: (...) -> None
```

Comment:
hmm, why the different syntax ?
```
def __init__(self, cache_dir, format_control, allowed_formats):
# type: (str, FormatControl, Set[str]) -> None
```
could work or am I missing something ?

Reply:
I think I overestimated how long the one-line annotation would be. Will fix!